timeline-chart/src/layer/time-graph-chart-cursors.ts

```diff
@@ -224,8 +224,8 @@ export class TimeGraphChartCursors extends TimeGraphChartLayer {
 
     update() {
         if (this.unitController.selectionRange) {
-            const firstCursorPosition = this.getPixel(this.unitController.selectionRange.start - this.unitController.viewRange.start);
-            const secondCursorPosition = this.getPixel(this.unitController.selectionRange.end - this.unitController.viewRange.start);
+            const firstCursorPosition = this.getWorldPixel(this.unitController.selectionRange.start);
```

Comment:
same comment as in timeline-chart/src/layer/time-graph-chart-selection-range.ts about APIs.

Reply:
This should also be resolved.